picture.py

```diff
@@ -0,0 +1,22 @@
+from PIL import Image
+
+
+def decipher():
+    """
+    https://stackoverflow.com/questions/56441769/iterate-over-all-pixels-to-check-which-pixels-are-white-and-which-are-black
+    Open file and decipher code.
+    :return: String decrypted from image.
+    """
+    black = 1
+
+    with Image.open("code.png").convert() as img:
```

Comment:
Don't use abbreviations in variable names (img) - 
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names